### ISSUE TITLE
Mention the codeblock formatting support for markdown

### ISSUE THREAD

**Issue by ikatyang** [prettier/prettier#4768]
Context: https://twitter.com/bohdan_kh/status/1012367393656483841

We should mention it in the markdown section of the landing page but not sure how to describe it, we do support formatting every supported language in the fenced code block.

https://github.com/prettier/prettier/blob/e765d31f8f847cf6b71e648509ab810667ad2271/website/data/languages.yml#L26-L31

**Comment by kellyk** [comment]
Does this mean that if you wrote some markdown like this:
```css
.sidebar { 
  background: red
}```
and then ran `yarn lint --fix` on the .md file, it would add a semicolon after red? I tried to get this working locally but couldn't get it working.

**Comment by ikatyang** [comment]
Yes, it should work. I'm not sure why it didn't work for you, could you open a new issue with repro steps?

**Prettier 1.14.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEADdYDOmA6UB0mAlgCZwBGAhgE4AEweVYA1gObUQCuUJSt1cEngC+edKhAAaEBAAOMItEzJQNDgHcACjQTKUlAG4RSUkOWqUWcGAGVZlolFbIY1TnGmPMcajE0XWAFtKZAAzSgAbb2kAK0wADwAhCytbSkC4ABlHODDI6JB7am9qZBBg6mYSCHUoU1lqRxgAdVIYAAtkAA4ABmkGiG9mi1kyhrgSg1zpAQBHTiIBf0ogkKRwqI8Qb0CiFzct4icIuABFTgh4PM3pGEpyVpIO5AAmW4siCMdWAGEIQOCZSg0GmIE43gAKvc9BtvMJhEA)
```sh
--parser markdown
```

**Input:**
````markdown
```css
.sidebar {
background: red
}
```
````

**Output:**
````markdown
```css
.sidebar {
  background: red;
}
```

````

**Comment by lydell** [comment]
This applies to more than code blocks in markdown:

- `<style>` and `<script>` tags in Vue and HTML (soon)
- `gql` and others in JS

**Comment by erick2014** [comment]
Hey guys I would like to work on this, but I'm not sure what should I fix here?

**Comment by lydell** [comment]
In this case, coding is the easy part. If we knew where to put this, we’d done it already :)

We’ve identified that we’d like to showcase Prettier’s cool multiparser feature but we aren’t sure exactly where to put it and what it would look like. Maybe around the “Works with the Tools You Use” section on the home page, maybe somewhere else.

If you’d like to experiment with this feel free to do so! Either with code and screen shots, or with some design tool, whatever you work best with.

**Comment by rukundob451** [comment]
@ikatyang was this closed or still open?

**Comment by etlovett** [comment]
@ikatyang @lydell What do you think about changing the "[Markdown](https://commonmark.org/), including [GFM](https://github.github.com/gfm/) and [MDX v1](https://mdxjs.com/)" bullet at https://prettier.io/docs/ to be something like one of the two options below? I'd be happy to make this change if we can agree on something!

* [Markdown](https://commonmark.org/), including [GFM](https://github.github.com/gfm/) and [MDX v1](https://mdxjs.com/) and language-specific formatting of embedded code blocks
* Markdown (both [CommonMark](https://commonmark.org/) and [GFM](https://github.github.com/gfm/)), including formatting of embedded code blocks and [JSX via MDX v1](https://mdxjs.com/)

**Comment by fisker** [comment]
It's not clear what to do, and it's already open for many years, I'll just close.
